src/deepsparse/yolo/utils.py

```diff
@@ -0,0 +1,386 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Helpers and Utilities for YOLO
+"""
+import itertools
+import time
+from typing import List, Optional, Tuple, Union
+
+import numpy
+import onnx
+import yaml
+
+import torch
+import torchvision
+
+
+try:
+    import cv2
+
+    cv2_error = None
+except ModuleNotFoundError as cv2_import_error:
+    cv2 = None
+    cv2_error = cv2_import_error
+
+COCO_CLASSES = [
```

Comment:
Maybe keeping this list in a separate file would be more elegant?

Reply:
> lgtm, good job. I would refactor some of the code, but I assume that those pieces which I do not like are copied from the original yolo codebase, so it does not make sense to beautify it - better to have it in the original form for reference, would you agree?

I agree a future PR would refactor that "copied" code out to use our own hosted YOLO wheel